etc/picongpu/bash-devServer-hzdr/mpiexec.tpl

```diff
@@ -44,13 +74,19 @@ umask 0027
 mkdir simOutput 2> /dev/null
 cd simOutput
 
-# test if cuda_memtest binary is available
-if [ -f !TBG_dstPath/input/bin/cuda_memtest ] ; then
-  mpiexec -am !TBG_dstPath/tbg/openib.conf --mca mpi_leave_pinned 0 -npernode !TBG_gpusPerNode -n !TBG_tasks !TBG_dstPath/input/bin/cuda_memtest.sh
+if [[ $VALIDATE_MODE == true ]]; then
+  echo "Validating PIConGPU call..."
+  !TBG_dstPath/input/bin/picongpu !TBG_author !TBG_programParams --validate | tee output
```

Comment:
Why do we need to validate in the submit script (generated by this `*.tpl`) if `tbg` already checks?

Reply:
Sorry - my fault - `tbg` just calls these lines